src/SourceBrowser/src/HtmlGenerator/SourceBrowser.nuspec

```diff
@@ -22,6 +22,8 @@
     <file src="*.exe" target="tools" />
     <file src="*.exe.config" target="tools" />
     <file src="*.dll" target="tools" />
+    <file src="BuildHost-net472\*.*" target="tools\BuildHost-net472" />
+    <file src="BuildHost-netcore\*.*" target="tools\BuildHost-netcore" />

```

Comment:
Do these need to be recursive?  If I check `microsoft.codeanalysis.workspaces.msbuild` it does have some subfolders.

Reply:
I don't know but this is from the upstream: https://github.com/KirillOsenkov/SourceBrowser/blob/main/src/HtmlGenerator/SourceBrowser.nuspec so maybe fixes should go there first? I don't think we care much here anyway about this nuspec, we don't publish/consume this as nuget package.